connectors/src/connectors/google_drive/temporal/file/handle_file_export.ts

```diff
@@ -124,6 +124,7 @@ export async function handleFileExport(
       connectorId,
       parents,
       tags: file.labels,
+      allowEmptySchema: true, // CSV files can be empty or with just one line, so we allow empty schemas
```

Comment:
what's the point of upserting csv without schema ?

Reply:
They seem ok not to index them, closing